help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -0,0 +1,159 @@
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Name of your source partner]**: Your customer/company name. Only supported Magnite Streaming clients will be available for selection.
+
+![destination configuration auth fields filled](../../assets/catalog/advertising/magnite/destination-realtime-config-auth-filled.png)
+
+Once done, click the Create button.
+
+![Optional governance policy and enforcement actions](../../assets/catalog/advertising/magnite/destination-realtime-config-grouping-policy.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>* To activate data, you need the **[!UICONTROL View destinations]**, **[!UICONTROL Activate destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+Once the destination has been created, you will be shown the audience activation flow. The following walks through how to activate audiences using the Real-Time destination.
+
+### Map attributes and identities {#map}
+
+The next step is mapping source identifiers to the Magnite device_id identifier.
+
+- You can add as many mappings as you need by selecting **[!UICONTROL Add new mapping]**.
+
+In this example using the Real-Time destination, we're mapping any rows that contain a generic deviceId source identifier to the Magnite device_id target field. When you\'re done, click Next.
```

Comment:
minor again, but we generally try to avoid phrasings with "we're doing something". so instead we could say:

This example using the Real-Time destination shows a row that contains a generic deviceId source identifier mapped to the Magnite device_id target field. When you're with the mappings, select **[!UICONTROL Next]**.

Reply:
Updated